python-rhsm.spec

```diff
@@ -53,6 +54,18 @@ A small library for communicating with the REST interface of a Red Hat Unified
 Entitlement Platform. This interface is used for the management of system
 entitlements, certificates, and access to content.
 
+
+%package certificates
+Summary: Certificates required to communicate with a Red Hat Unified Entitlement Platform
+Group: Development/Libraries
+Version: 1.0.0
```

Comment:
@awood fixed cause I think you know better, but given that this package should rarely change, it seems inefficient to update the package every time python-rhsm is updated even though it would be a no-op.